packages/wonder-blocks-clickable/src/components/__tests__/clickable.test.js

```diff
@@ -452,6 +454,56 @@ describe("Clickable", () => {
         );
     });
 
+    test("should add aria-disabled if disabled is set", () => {
+        // Arrange
+
+        // Act
+        render(
+            <Clickable testId="button" disabled={true}>
+                {(eventState) => <h1>Click Me!</h1>}
+            </Clickable>,
+        );
+
+        const button = screen.getByTestId("button");
+
+        // Assert
+        expect(button).toHaveAttribute("aria-disabled", "true");
+    });
+
+    test("allow keyboard navigation when disabled is set", () => {
+        // Arrange
+        render(
+            <Clickable testId="button" disabled={true}>
+                {(eventState) => <h1>Click Me!</h1>}
+            </Clickable>,
+        );
+
+        // Act
+        userEvent.tab();
```

Comment:
Is the button receiving focus because of this `userEvent.tab()`? It is possible that the button could be receiving focus because it's the only thing being rendered here? Sorry, I'm not entirely sure how this works but I'm just wondering because I struggled to test with `tab()` in the past.

Reply:
That's an excellent question 🤔. By default, RTL gives focus to the `body` element, so this test is kinda tricky. I'll change it to be more explicit about this, and will probably add a couple of extra focusable elements (other buttons that are not `Clickable` elements) to make the test more robust.

Reply:
I updated the test to make it clearer. Let me know what you think.